llvm/unittests/CMakeLists.txt

```diff
@@ -14,6 +14,11 @@ function(add_llvm_target_unittest test_dir_name)
   add_llvm_unittest(${test_dir_name} DISABLE_LLVM_LINK_LLVM_DYLIB ${ARGN})
 endfunction()
 
+# GCC may emit false positive warnings against LLVM's style guide.
```

Comment:
Can you expand the comment to note that this is because of the gtest macro expansion.

Reply:
Sure. Done. Thanks @teresajohnson !